tests/everest/test_detached.py

```diff
@@ -227,37 +222,30 @@ def test_generate_queue_options_no_config():
 
 
 @pytest.mark.parametrize(
-    "queue_options, expected_result",
+    "queue_class, queue_kwargs",
     [
         (
-            {"partition": "ever_opt_1", "name": "slurm"},
-            SlurmQueueOptions(max_running=1, partition="ever_opt_1"),
+            SlurmQueueOptions,
+            {"name": "slurm", "partition": "ever_opt_1", "max_running": 1},
         ),
+        (LsfQueueOptions, {"name": "lsf", "lsf_queue": "ever_opt_1", "max_running": 1}),
         (
-            {"lsf_queue": "ever_opt_1", "name": "lsf"},
-            LsfQueueOptions(
-                max_running=1,
-                lsf_queue="ever_opt_1",
-            ),
-        ),
-        (
-            {
-                "name": "torque",
-                "keep_qsub_output": "1",
-            },
-            TorqueQueueOptions(max_running=1, keep_qsub_output=True),
+            TorqueQueueOptions,
+            {"name": "torque", "keep_qsub_output": True, "max_running": 1},
         ),
     ],
 )
 def test_generate_queue_options_use_simulator_values(
```

Comment:
I think "server" is missing from this test name. It seems we are testing the feature of copying simulator settings to server settingsl